CONTRIBUTING.md

```diff
@@ -118,30 +118,49 @@ Once you are satisfied with your PR you should ask someone to review it. Before
 
 ## Release protocol
 
-Once you are ready to publish a release, branch off from `dev`
+### Before the release
+
+1. Create a release branch by branching off from `dev`
     ```bash
     git checkout -b release/vX.Y.Z dev
     ```
-For meaning of X, Y and Z version numbers, please refer to this [semantic versioning guidelines](https://semver.org/spec/v2.0.0.html).
-
-In this branch, you should normally only update the version number in the `src/multi-vector-simulator/version.py` and in the `CHANGELOG.md` files, respecting the indicated formats. Commit the first one with "Bump version number" as commit message.
-
-Your `CHANGELOG.md` file could look like this before the release
-```
-## [unreleased]
-
-### Added
-- feature 1
-- feature 2
-### Changed 
-- thing 1
-- thing 2
-### Removed
-- some stuff
-```
-
-Simply replace `unreleased` by `X.Y.Z` and add the date of release in [ISO format](https://xkcd.com/1179/), then add the structure for a new `unreleased` version
-
+    For meaning of X, Y and Z version numbers, please refer to this [semantic versioning guidelines](https://semver.org/spec/v2.0.0.html).
+2. In your release branch, update the version number in [`src/multi-vector-simulator/version.py`](https://github.com/rl-institut/multi-vector-simulator/blob/dev/src/multi_vector_simulator/version.py) (commit message: "Bump version number") and
+in [`CHANGELOG.md`](https://github.com/rl-institut/multi-vector-simulator/blob/dev/CHANGELOG.md) in the in the format indicated under 1.
+Attention: Please work with release candidates (add `rc1` to the `version_num`, for example `v0.5.0rc1`) before the actual release, as a release with a specific version number can only be done once.
+3. Add the date of release in the `Changelog.md` in [ISO format](https://xkcd.com/1179/).
```

Comment:
Maybe "replace the changelog header `[Unreleased]` with the number and date of the release `[Version] - YYYY-MM-DD`"?